docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+     val ptextent = ptfToExtent(ptf)
+      val gridBounds = ld.mapTransform(ptextent)
+      for ((c, r) <- gridBounds.coords;
+           if r < tl.totalRows;
+           if c < tl.totalCols) yield (SpatialKey(c,r), ptf)
+    }
+
+    val keyfeatures = pts.flatMap(ptfToSpatialKey).groupBy(_._1).map {
+      case (sk,v) => (sk,v.unzip._2)
+    }: Map[SpatialKey,List[PointFeature[Double]]]
+```
+
+Now, all the subtiles may be generated in the same fashion as the monolithic tile case above.
+
+```scala
+    val keytiles = keyfeatures.map { 
+      case (sk, pfs) => (sk, pfs.kernelDensity (kern,
+                                                RasterExtent(ld.mapTransform(sk),
+                                                             tl.tileDimensions._1,
+                                                             tl.tileDimensions._2))) 
+    }
+```
+
+Finally, it is necessary to combine the results.  Note that, in order to produce a 700x400 tile that is identical to the simpler, non-tiled case, every SpatialKey must be represented in the map, or the result may not span the full range.  This is only necessary if it is important to generate a tile that spans the full extent.
+
+```scala
+  import geotrellis.spark.stitch.TileLayoutStitcher
+
+  val tileList = 
+      for { r <- 0 until ld.layoutRows
+          ; c <- 0 until ld.layoutCols
+          } yield {
+            val k = SpatialKey(c,r)
+            (k, keytiles.getOrElse(k, IntArrayTile.empty(tl.tileCols, 
+                                                         tl.tileRows)))
+          }
+    val stitched = TileLayoutStitcher.stitch(tileList)._1
+```
+
+It is now the case that `stitched` is identical to `kde.`
+
+### Distributing Computation via Apache Spark ###
+
+As mentioned, the reason for introducing this more complicated version of kernel density estimation was to enable distributed processing of the kernel stamping process.  Each subtile could potentially be handled by a different node in a cluster, which would make sense if the dataset in question were, say, the location of individual trees, requiring a huge amount of working memory.  By breaking the domain into smaller pieces, we can exploit the distributed framework provided by Apache Spark to spread the task to a number of machines.  This will also provide fault tolerant, rapid data processing for real-time and/or web-based applications.  Spark is much too big a topic to discuss in any detail here, so the curious reader is advised to search the web for more information.
+
+Our concern falls on how to write code to exploit the structures provided by Spark, specifically *Resilient Distributed Datasets*, or RDDs.  An RDD is a distributed collection, with all of the usual features of a collection---i.e., mapping, folding, reducing.  In order to create an RDD, one will call the `parallelize()` method on a `SparkContext` object, which can be generated by the following set of statements.
```

Comment:
RDD's actually don't have a fold, which is a key difference (fold requires sequential processing). Instead they have `aggregate`. There actually is a "fold" method, but that's just a reduce with a seed value. I would just maybe change the wording to not set up the user expectation that RDDs have all the features of collections...`with many of the useful features of collections -- e.g. map, reduce -- as well as distributed versions of other types of sequential collection methods -- e.g. aggregate, which is a distributed version of something like foldLeft`.